.github/actions/setup-spack-stack/action.yaml

```diff
@@ -118,15 +122,23 @@ runs:
           export FC=gfortran-10
           export CXX=g++-10
           export FFLAGS="-fallow-argument-mismatch"
+          export FCFLAGS="-fallow-argument-mismatch"
```

Comment:
`mpich` wants both `FFLAGS` and `FCFLAGS` (unlike `openmpi`, which is happy with just `FFLAGS`).